docs/swagger.yaml

```diff
@@ -204,6 +204,44 @@ paths:
                         type: string
                         description: message
                         example: Password reset email sent
+  /auth/password/{token}:
+    patch:
+      tags: 
+        - user
+      operationId: resetPassword
+      summary: Reset password with token
+      description: Reset password with token as a parameter
+      requestBody:
+        description: Reset password
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                password:
```

Comment:
I think it should be `newPassword` right?

Reply:
yeah. Silly mistake